install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/tasks/build_securedrop_app_code_deb.yml

```diff
@@ -16,38 +23,9 @@
 
 - include: sass.yml
 
-- name: Create pip wheel archive for Debian package requirements.
+- name: Install SecureDrop Python requirements.
   shell: |
-    pip wheel \
-      --no-binary :all: \
-      --require-hashes \
-      --requirement {{ securedrop_pip_requirements }} \
-      --wheel-dir {{ securedrop_wheelhouse }} 2>&1 | tee /tmp/w.out
-      ! grep -i --quiet 'Failed to build' /tmp/w.out
-  register: wheel_build_output
-  tags: pip
-
-- name: Ensure source hash sums matched at wheel build-time
-  fail: msg="Source hash sum mismatch, build cannot continue."
-  failed_when: "'THESE PACKAGES DO NOT MATCH THE HASHES' in wheel_build_output.stdout"
-
-# Here we need to regenerate a new securedrop-app-code-requirements.txt file
-# *without* hashes to bundle in the .deb package, for reasons that follow:
-# If requirements hashes are in the requirments file, pip will automatically
-# use hash checking mode to install the dependencies (which is done in postinst
-# of securedrop-app-code package).
-# dpkg-buildpackage modifies the wheel archive such that the hashes of those
-# built wheels change. Since the deb package itself is signed (providing
-# integrity, we can defer to a later time.
-- name: Install built pip wheels for SecureDrop.
-  shell: |
-    pip install {{ securedrop_wheelhouse }}/*.whl
-  tags:
-    - pip
-
-- name: Create new requirements based on build/installed wheels without hashes
-  shell:
-    pip freeze > {{ securedrop_pip_requirements_generated }}
+    pip3 install -r {{ securedrop_pip_requirements }}
```

Comment:
We should not install any dependency as part of packaging. This has to be handled by the `dh-virtualenv`.

Reply:
This is installing our requirements in the build container, so we can do things like run `i18n_tool.py`. I don't believe this affects `dh-virtualenv`, but if I'm wrong, please set me straight.

Reply:
Yes, we should not run `i18_tool.py` on the build step, everything should be baked and ready inside of the release tarball.

Reply:
It has to be run at some point when building this package, or we won't get the .mo files. I've modified the playbook to ensure that the .mo files are added to the sdist tarball, and eventually still make it into the .deb.